### ISSUE TITLE
Bug: Missing variant with the new version 2-alpha.25

### ISSUE THREAD

**Issue by hassansaei** [hassansaei/VNtyper#50]
A 23bp insertion variant was successfully identified using VNtyper_v1.3, but it is now being missed when using the online version or cli processing from BAM and FASTQ files with the --fast-mode option. We need to investigate this issue and resolve it ASAP. 

It also failed when running in the normal mode without the --fast-mode option. 

[pipeline.log](https://github.com/user-attachments/files/18207998/pipeline.log)

Update: It worked with slow mode, and the issue was the number of threads I assigned.

**Comment by berntpopp** [comment]
Do you have a sample file?
Subset BAM (generated with the online tool) might also be okay.

**Comment by hassansaei** [comment]
I shared with you the BAM file! I think it is coming back to the Kestrel upgrade.

**Comment by berntpopp** [comment]
Relates to #52

**Comment by berntpopp** [comment]
I will downgrade to 1.0.1 in #52

**Comment by berntpopp** [comment]
@hassansaei : Can you try version > 2.0.0-alpha.37 please? If it finds the variant, please close #52

**Comment by hassansaei** [comment]
I tested version 2.0.0-alpha.37 with a BAM file, and it successfully detected the variant.

However, there is an issue with the directory assignment for index files during alignment. In the configuration file, we specified the path as reference/alignment/chr1.hg19.fa, but during the build process, the genome sequence and index files are saved in /opt/vntyper/references.

https://app.warp.dev/block/aGBIjXc9SVGMZ6r60AijAm  

The correct path for the index files should be:
references/reference/alignment/chr1.hg19.fa.

or we can specify "/opt/vntyper/" as the output_dir during the build process.
